Grant Application - Reference blockchain system.md

```diff
@@ -0,0 +1,286 @@
+
+We believe in our system, creators or owners of each work will choose to open and share their works because with our incentive mechanism, opening and sharing can give them more return than closure and monopoly. Everyone can have the same rights and freedom as them to develop and improve their works easily on our fully composable and interoperable system. In this way, we can form unlimited, permissionless, and boundless Internet-scale online collaboration to build a fully decentralized and incentive Web3 made by everyone, and for everyone without private monopoly or privileges.
+
+
+### Project Details
+
+In Web1 users read online content and use online services via protocols like SMTP, HTTP, and RSS. Protocols are decentralized that no single entity fully owns and controls them, but also resulted in no one having incentives to provide better public online services. Therefore, although being widely used, they still remain primitive unchanged until now. 
+
+In Web2 users are not just consumers, but also become service providers to create online content and provide online service on platforms. Those platforms are owned and controlled privately, which give their owners incentives to manage and improve online services to the public better continuously. In this way Web2 online services have higher quality than Web1, but such centralization also causes problems as service managers maximize self-interests at the expense of others in public.
+
+Therefore Web3 aims at solving the problem of non-incentive in Web1, and centralization in Web2 to make online services both decentralized and incentive. In Web3, online public services should be maintained and managed not by any centralized entity in private, but by users in public collectively. Users will not only consume and provide online services, but also manage them together to protect their own digital rights, freedom and assets. 
+
+Some Web3 projects try to break the monopoly among different platforms, and give some rights of self-governance to the community via DAO, but still far from the goal above. A central team like the founding company still has more privileges and plays a more vital role than others in determining the development and management of its project, and they predefine and limit the self-governance of the DAO, making it not so useful. To participate in DAO, users need to buy and stake governance tokens, which sets a high threshold and causes inequality. The distribution of governance tokens is often uneven, making DAO not so decentralized and representative. The users who don’t have the majority of votes have to compromise. 
```

Comment:
That is a very narrow view of what DAOs today are. Whether tokens are required for participation and how power is distributed among its members is a design decision, not a technical limitation. There are DAO platforms in existence that let you design and deploy DAOs with whatever mechanics you desire.

Reply:
Yes we are exploring a new mechanism of DAO besides staking and voting. DAO could be designed in any way without technical limitation, so we are designing a new way and we think it could be better than any other existing ways.